css/src/components/breadcrumbs.scss

```diff
@@ -0,0 +1,30 @@
+$breadcrumb-font-size: $font-size-8 !default;
+$breadcrumb-spacing-horizontal: 0.25em !default;
+$breadcrumb-spacing-vertical: 0.375em !default;
+$breadcrumb-separator: '\00200B/' !default;
+
+.breadcrumbs {
+	font-size: $breadcrumb-font-size;
```

Comment:
I think font-size should be determined externally, via utility, but I don't feel strongly about that.

Reply:
If anyone who uses Atlas would want to overwrite the value they would be able to set up the `$breadcrumb-font-size` to whatever fits their need. So I think it should stay as is.